biz.aQute.bndlib/src/aQute/bnd/differ/DiffPluginImpl.java

```diff
@@ -100,9 +100,8 @@ public Tree tree(File newer) throws Exception {
 	 * @see aQute.bnd.service.diff.Differ#tree(aQute.bnd.osgi.Jar)
 	 */
 	public Tree tree(Jar newer) throws Exception {
-		Analyzer anewer = new Analyzer();
+		Analyzer anewer = new Analyzer(newer);
 		try {
-			anewer.setJar(newer);
 			return tree(anewer);
 		} finally {
 			anewer.setJar((Jar) null);
```

Comment:
When the analyzer is closed, the Jar will be closed because setJar calls addClose.